widget/embedded/src/containers/Wallets/Wallets.tsx

```diff
@@ -99,6 +107,16 @@ function Main(props: PropsWithChildren<PropTypes>) {
       props.onUpdateState(type, event, value, state, meta);
     }
   };
+  const reloadWallet = useCallback(
+    () => connectWallet(accounts, tokens),
+    [accounts]
```

Comment:
You need to add `tokens` to callback dependency as well.

Reply:
nit: I think using `useCallback` here doesn't add any value, because all the codes inside the function are references so you won't save anything here.

Reply:
fixed.